lib/cdo/contact_rollups_validation.rb

```diff
@@ -0,0 +1,308 @@
+              WHERE courses_facilitated IS NOT NULL
+              AND Roles NOT LIKE '%Facilitator%'",
+      min: 400,
+      max: 40000
+    },
+    {
+      name: "Count of contacts with professional learning enrollment",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE professional_learning_enrolled IS NOT NULL",
+      min: 13000,
+      max: 100000
+    },
+    {
+      name: "Count of contacts with professional learning attended",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE professional_learning_attended IS NOT NULL",
+      min: 13000,
+      max: 100000
+    },
+    {
+      name: "Count of contacts with HOC organizer years",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE hoc_organizer_years IS NOT NULL",
+      min: 375000,
+      max: 750000
+    },
+    {
+      name: "Count of contacts with grades taught",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE grades_taught IS NOT NULL",
+      min: 200000,
+      max: 400000
+    },
+    {
+      name: "Check that all contacts with grades taught are "\
+            "teachers",
+      query: "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE grades_taught IS NOT NULL
+              AND Roles NOT LIKE '%Teacher%'",
+      min: 0,
+      max: 0
+    },
+    {
+      name: "Count of contacts with ages taught",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE ages_taught IS NOT NULL",
+      min: 190000,
+      max: 400000
+    },
+    {
+      name: "Check that all contacts with ages taught are "\
+            "teachers",
+      query: "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE ages_taught IS NOT NULL
+              AND Roles NOT LIKE '%Teacher%'",
+      min: 0,
+      max: 0
+    },
+    {
+      name: "Count of contacts with forms submitted",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE forms_submitted IS NOT NULL",
+      min: 2_600_000,
+      max: 5_000_000
+    },
+    {
+      name: "Check that all contacts with forms submitted are "\
+            "form submitters",
+      query: "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE forms_submitted IS NOT NULL
+              AND Roles NOT LIKE '%Form Submitter%'",
+      min: 0,
+      max: 0
+    },
+    {
+      name: "Count of contacts with form roles",
+      query:  "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE form_roles IS NOT NULL",
+      min: 1_750_000,
+      max: 3_500_000
+    },
+    {
+      name: "Check that all contacts with form roles are "\
+            "form submitters",
+      query: "SELECT COUNT(*) FROM contact_rollups_daily
+              WHERE form_roles IS NOT NULL
+              AND Roles NOT LIKE '%Form Submitter%'",
+      min: 0,
+      max: 0
+    },
+  ].freeze
+
+  def self.validate_contact_rollups
```

Comment:
Please document (in YARD) what this does and what it returns. Also the methods below.